schema/encoding/jsonschema/benchmark_test.go

```diff
@@ -38,15 +39,25 @@ func BenchmarkEncoder(b *testing.B) {
 			},
 		},
 		{
-			Name:   `Schema=Student`,
+			Name:   `Schema=Simple`,
 			Schema: studentSchema,
 		},
+		{
+			Name:   `Schema=Complex1`,
+			Schema: requestSchema,
+		},
+		{
+			Name:   `Schema=Complex2`,
+			Schema: responseSchema,
+		},
 	}
+	buf := bytes.NewBuffer(make([]byte, 65535))
```

Comment:
I think relying on [ioutil.Discard](https://golang.org/pkg/io/ioutil/#pkg-variables) is probably the best option btw.